apps/web-staking/src/app/components/redeem/RedeemComponent.tsx

```diff
@@ -20,13 +16,13 @@ import AgreeModalComponent from "../modal/AgreeModalComponents";
 import RedeemWarning from "@/app/components/redeem/RedeemWarning";
 import { PrimaryButton, StakingInput } from "@/app/components/ui";
 import { ConnectButton } from "@/app/components/ui/buttons";
+import { RedemptionFactor } from "@/services/redemptions.service";
 
 export default function RedeemComponent() {
-	const { open, close } = useWeb3Modal();
-	const { address, chainId } = useAccount();
+	const { open } = useWeb3Modal();
```

Comment:
I can see a merge conflict when switching to reown for this refactor ^^

Reply:
Is there something that should be done here to mitigate that?